src/app/screens/__tests__/Login.test.tsx

```diff
@@ -56,18 +56,4 @@ describe('Login', () => {
 
     expect(mockStore.getActions()).toContainEqual(login('password'));
   });
-
-  it('dispatches login when pressing enter', async () => {
```

Comment:
Why would we want to remove this test?

Reply:
Because this is handled with the default browser functionality now, not with React handlers. Firing an event does not submit the form anymore.